### DISCUSSION TITLE
Public capacity error in LND

### DISCUSSION THREAD

**Discussion by amtexugo** [cryptosharks131/lndg#321] [General]
Hello friends!
There was an error in the public capacity of my channel. See in LND it is 10 million and in Amboss is 79 million which is correct. What could have happened?...Thank you

https://amboss.space/node/022db2e3a531bb8f85d258c134ba013b7d17bc04d6984efebf9c1f0f63e9c0f596

![Capturar_2023_08_23_13_38_43_337](https://github.com/cryptosharks131/lndg/assets/127704297/6cb827e1-696d-47e6-a85a-313d99edadd2)

![Capturar_2023_08_23_13_40_07_941](https://github.com/cryptosharks131/lndg/assets/127704297/745e04d4-7138-4cc9-a21d-6aa15f498cf7)

**Comment by cryptosharks131** [comment]
This should be resolved in `v1.8.0`.
